tasmota/tasmota_xx2c_global/xx2c_interface.ino

```diff
@@ -15,7 +15,7 @@
 
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+*/ 
```

Comment:
Remove these spaces as other than that no changes are made to this file.

Ditto other locations.